op-program/client/l2/engineapi/precompiles.go

```diff
@@ -50,17 +50,17 @@ type PrecompileOracle interface {
 }
 
 func CreatePrecompileOverrides(precompileOracle PrecompileOracle) vm.PrecompileOverrides {
-	return func(rules params.Rules, orig vm.PrecompiledContract, address common.Address) (vm.PrecompiledContract, bool) {
+	return func(rules params.Rules, orig vm.PrecompiledContract, address common.Address) vm.PrecompiledContract {
```

Comment:
Since this should never add a new precompile, could we add a defensive:
```
if orig == nil { return nil }
```

Otherwise this will enable the kzgPointEvaluation precompile prior to the fork it was original added to. While no chains will actually hit that case with new precompiles it could easily introduce problems.

Reply:
great catch, will do this